src/framework/gen.js

```diff
@@ -561,6 +561,83 @@ $body-font: ${model.font};
     .LayoutScrolling--maincontainer {
         background-color: $background-medium-color;
     }
+
+    .Nav-LayoutBox {
+        min-width: 1905px;
+        min-height: 415px;
+    }
+
+    .Gtk-button-back {
+        color: $accent-light-color;
+        font-family: $title-font;
+        font-size: 25px;
+        margin-left: 95px;
+        margin-top: 65px;
+        min-height: 150px;
+        min-width: 150px;
+    }
+
+    .ContentGroup--navcontent {
+        background-color: #3f3532;
+        background-image:
+            url('ekn:///eebc9b8230883b63fccace14f32c594a661cb2da'),
+            url('ekn:///88ebc9b8230b63fccace14f32c594a661cb2da3e');
+        background-repeat: no-repeat, repeat-y;
+        background-position: center top, center 17px;
+        min-height: 100px;
+        min-width: 100px;
+        padding-bottom: 65px;
+        padding-top: 65px;
+
+        .CardHackdexArticle {
+            min-height: 290px;
+            min-width: 240px;
+
+            frame {
+                background-position: center -5px;
+                background-size: 240px;
+            }
+
+            .CardHackdex__title {
+                background-position: center -362px;
+                color: transparent;
+                font-size: ${model.font_size * 4.5}px;
+                margin: 0 2px;
+                padding: 10px;
+            }
+
+            &:hover {
+                .CardHackdex__title {
+                    color: $primary-light-color;
+                }
+
+                frame {
+                    background-position: center -377px;
+                    border: 7px solid white;
+                }
+            }
+
+            &.CardHackdex-0:hover .CardHackdex__title {
```

Comment:
I would like to avoid setting the background for these cards in the CSS, because if we change the sorting order of the cards in the hacking panel then we'll get the wrong backgrounds, as the names will be sorted differently but the backgrounds will be tied to the physical position of the card on screen. The content metadata already provides a "thumbnail URI" property for each card, can we use that?

Reply:
@ptomato I have it clear now and agree on avoiding using CSS for this cards. The metadata already provides the "thumbnail URI" property for each card, we actually can use that.  I`ll be working on it during the week and deliver as soon as possible.